docs/doxygen/pages.txt

```diff
@@ -1,4 +1,9 @@
 /**
 @mainpage Overview
 @brief The AWS IoT Device SDK for Embedded C provides demonstration code and integration tests for a collection of libraries.
+
+The following libraries are part this SDK:
```

Comment:
Should this be in a `@section` or `@subpage` so that it shows up in the sidebar?

Reply:
I didn;t think it needed to be there. But I guess it will look nice